pkg/controller/common/common.go

```diff
@@ -0,0 +1,11 @@
+package common
+
+import "github.com/konveyor/mig-controller/pkg/settings"
+
+func IsInSingletonNamespace(namespace string) bool {
```

Comment:
Perhaps we can find a more descriptive name than "Singleton".  Perhaps "Privileged"?

Reply:
Privileged sounds much better to me. @dymurray?

Reply:
"Singleton" generally refers to an object for which only one is able to exist its type/class. There isn't generally a notion of "privileged" there. If privileges are relevent, then I'd say "PrivilegedNamespace" makes more sense.

Reply:
@mansam yes `privileged` sounds good to me. Or `admin`.